theme/views.py

```diff
@@ -175,16 +175,18 @@ def rating(request):
     rating_form = RatingForm(request, obj, post_data)
     if rating_form.is_valid():
         rating_form.save()
+        return response
         if request.is_ajax():
             # Reload the object and return the rating fields as json.
             obj = obj.__class__.objects.get(id=obj.id)
             rating_name = obj.get_ratingfield_name()
             json = {}
             for f in ("average", "count", "sum"):
                 json["rating_" + f] = getattr(obj, "%s_%s" % (rating_name, f))
-            response = HttpResponse(dumps(json))
+            return HttpResponse(dumps(json))
```

Comment:
Adding line 178 makes lines 179-186 useless which should be deleted.

Reply:
Or line 178 should not be added?

Reply:
Looks to me line 178 should not be added, since otherwise, ```rating_sum``` which is used by the template will not be part of the response.

Reply:
I removed the function block of 178. It was also setting the cookie that I believe is not required at this moment.